File: MODELJUNIT/src/implementation/profonline/models/classes.php
<?php

class Classes {
    private $listPlan = [ '0' => ['period' => 20192, 'classId' => 1, 'closePlanOk' => 'true' ], '1' => ['period' => 20192, 'classId' => 2, 'closePlanOk' => 'false' ] ];

    public function validateChooseClass( $period, $classId ){
        //var_dump($listPlan);die;

        foreach ($this->listPlan as $key => $value) {
            // $arr[3] será atualizado com cada valor de $arr...
            if( $period == $value['period'] && $classId == $value['classId'] ){
                echo $value['closePlanOk'] . "";die;
            }
        }
        echo "Not Found";
    
    }
}

//incluir buscar na lista toda, e retornar o closePlanOk.

?>

File: Sd2Stubs/Files/Classes.java
public class Classes {
	private String period = "20192";  
	private String classId = "1";  

	public String getPeriod() {
		return period;
	}
	
	public String getClassId() {
		return classId;
	}  

	public void updateCloseClass(String period, String classId) {
	}
	
	public boolean validateChooseClass(String period, String classId) {
		if (this.period.equals(period) && this.classId.equals(classId)){
			return true;
		}
		else{
			return false;
		}
	}
}
